components/compositing/compositor.rs

```diff
@@ -99,6 +102,15 @@ impl FrameTreeId {
     }
 }
 
+#[derive(Debug)]
+struct CompositorHitTestResult {
+    pipeline_id: PipelineId,
+    point_in_viewport: euclid::default::Point2D<f32>,
+    point_relative_to_item: euclid::default::Point2D<f32>,
+    node: UntrustedNodeAddress,
+    cursor: Option<u16>,
```

Comment:
Do we need the raw u16 cursor value here to keep this generic (and not the result of Cursor::from_u8)? I'm not very familiar with this code, so it is not obvious to me.

Reply:
You're totally correct. I've updated this change so that `Cursor` is used everywhere instead of converting the value to `u16`. This simplifies the code a bit.